nosekatreport/decorators.py

```diff
@@ -166,137 +149,6 @@ class TestGeneric(unittest.TestCase):
     """
     return grand_decorator('generic_test', *args)
 
-def instrument_bc8n856M4k(*args):
```

Comment:
Has it been confirmed that these instrument decorators are not used by CBF?

Reply:
CBF uses this branch: https://github.com/ska-sa/nosekatreport/tree/karoocbf
Any further development is being done in the karoocbf branch.